lib/trento/databases/events/legacy/database_deregistered.ex

```diff
@@ -5,7 +5,7 @@ defmodule Trento.SapSystems.Events.DatabaseDeregistered do
 
   use Trento.Support.Event
 
-  defevent do
+  defevent superseded_by: Trento.Databases.Events.DatabaseDeregistered do
```

Comment:
Why do we need to add superseded_by again?

Reply:
Because the name of the module is changed from `Trento.SapSystem.Events` to `Trento.Databases.Events`